src/workflow.tsx

```diff
@@ -5,9 +5,7 @@ interface WorkflowProps {
   name: string;
 }
 
-const Workflow: React.SFC<WorkflowProps> = ({
-  name,
-}: WorkflowProps): JSX.Element => (
```

Comment:
`WorkflowProps` after `{ name }` is not needed because ` React.FunctionComponent<WorkflowProps>` declaration makes the type of the params inferred.

Reply:
👍